x-pack/elastic-agent/pkg/agent/application/upgrade/service_windows.go

```diff
@@ -63,9 +62,7 @@ func (p *pidProvider) PID(ctx context.Context) (int, error) {
 }
 
 func invokeCmd(topPath string) *exec.Cmd {
-	homeExePath := filepath.Join(topPath, agentName)
-
-	cmd := exec.Command(homeExePath, watcherSubcommand,
+	cmd := exec.Command(paths.TopBinaryPath(), watcherSubcommand,
```

Comment:
`topPath` isn't used anymore as an argument to this function

Reply:
Fixed.